.github/workflows/ci.yml

```
@@ -0,0 +1,60 @@
+name: ci
+
+on: [pull_request, push]
```

I don't think we want this running on every push.

```suggestion
on: [pull_request]
```

---

Done